### PR TITLE
:mortar_board: Lab10

### PR DESCRIPTION
**Pull request by jameshughes89**
### What
Create a lab 10 focusing on the "simpler" sorting algorithms 

### Why
As of now I am thinking that lab 11 will focus on quick and merge sort, so haveing a baseline is cool.

### Testing
Built locally

### Additional Notes
1. Ended up going with arrays, but added a note that they could do it with bags if they want. This may be useless though. 
2. I mayyyyyyyyyyyy make another topic lab 10, and then make this one 11, but I'll see.

### CODE REVIEW

**Comment by jameshughes89** [comment]
Next lab I will do in chunks --- this one is now too big and i do not like

File: src/site/lab10.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+        printArray(toSort);
+        System.out.println((endTime - startTime)/1000000);  // Runtime in ms
+    }
+
+    public static int[] createUnsortedArray(int n, int maxValue){
+        Random randNumGen = new Random();
+        int[]  toSort= new int[n];
+        for(int i = 0; i < n; ++i){
+            toSort[i] = randNumGen.nextInt(maxValue);
```
Comment (twentylemon) [inline src/site/lab10.rst:52]: I know `Random` is the way to go, but the import here makes it impossible to copy/paste. [Math.random](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Math.html#random()) is in `java.lang`, so it doesn't require an import.

File: src/site/lab10.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        printArray(toSort);
+
+        double startTime = System.nanoTime();
+        bubbleSort(toSort);
+        //insertionSort(toSort);
+        //selectionSort(toSort);
+        double endTime = System.nanoTime();
+
+        printArray(toSort);
+        System.out.println((endTime - startTime)/1000000);  // Runtime in ms
```
Comment (twentylemon) [inline src/site/lab10.rst:45]: Move the comment into the string being output instead. `println("runtime: " + (x) + " ms")`

File: src/site/lab10.rst (hunk 1 above)
Comment (jameshughes89) [reply src/site/lab10.rst:52]: I actually originally had that, but thought it was uglier, so I went with `Random`. That said, seeing another vote for `Math.random(); `, I'll revert back.

**Review by twentylemon — APPROVED** [review]

**Pull request merged**